www/src/_layouts/docs.njk

```diff
@@ -23,17 +24,17 @@ styles:
   <main class="docs__container">
     <ul class="docs__side-nav" role="list">
       {%- for navLink in navLinksSorted -%}
-      <li>
-        <a
+        <li>
+          <a
           href="{{navLink.href}}"
           {% if navLink.href === page.url %}aria-current="page"{% endif %}
         >
-          {{navLink.title}}
-        </a>
-      </li>
+            {{navLink.title}}
+          </a>
+        </li>
       {%- endfor -%}
     </ul>
-    <article class="docs__container__content">
+    <article class="docs__container__content" id="main-content">
```

Comment:
Howdy- I left a comment a while back that I think got a little lost, but the skip link target should have a `tabindex="-1"` to ensure that, especially in mobile browsers, focus actually moves here.